mxcube3/routes/SampleCentring.py

```diff
@@ -30,12 +30,8 @@ def init_signals():
     callback method.
     """
     for signal in signals.microdiffSignals:
-        if signal in signals.task_signals:
-            mxcube.diffractometer.connect(mxcube.diffractometer,
-                                          signal, signals.task_event_callback)
```

Comment:
The list of signals might need to be cleaned up a bit and I might have made a mistake here, but I cant really find were the signal warning is emitted, also it seems to be present in other signals lists as well which makes me wonder a little bit